src/runtime/account.ts

```diff
@@ -45,23 +70,97 @@ export class Account {
   }
 
   get accountId(): string {
-    return this.najAccount.accountId;
+    return this._accountId;
+  }
+
+  get prefix(): string {
+    return this.levelUp ? this.accountId.replace(`.${this.levelUp}`, "") : this.accountId;
   }
 
   async balance(): Promise<AccountBalance> {
     return this.najAccount.getAccountBalance() as Promise<AccountBalance>;
   }
 
+  createTransaction(receiver: Account | string): Transaction {
+    return new RuntimeTransaction(this.runtime, this, receiver);
```

Comment:
I wonder if the cast to `string` should happen here, so `Transaction` doesn't need to know anything about `accountId`

Reply:
I think it's fine since there is already a notion of the 'Account' class in `Transaction` and it doesn't require a runtime.  So it's still handy.